assets/css/style.css

```
@@ -5,16 +5,86 @@
 
 body {
   margin: 0 auto;
+  }
+
+body, html, ul, ol, li, p, h1, h2, h3, h4 {
+  margin: 0px;
+  padding: 0px;
+}
+
+div {
+  box-sizing: border-box;
+  }
+
+.wrapper {
   max-width: 1600px;
+  margin: 0 auto;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 36px;
+
 }
 
 .header-section {
   padding-left: 175px;
   padding-right: 175px;
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  height: 110px;
+
 }
 
 .header-title {
   color: #fff;
   font-size: 36px;
   text-transform: uppercase;
+  font-family: 'Yeseva One', 'Arial', 'Times New Roman', serif;
+}
+
+.menu-bar,
+.socials {
+ align-items: center;
+ display: flex;
+ height: 21px;
+ list-style: none;
 }
+
+.intro-container {
+  background-image: url(/assets/images/background-intro.png);
+  background-position: 50% 0px;
+  background-repeat: no-repeat;
+}
+
+.nav-bar {
+  display: flex;
+}
+
+.menu-item {
+  color: white;
+  font-size: 21px;
+  text-transform: capitalize;
+  padding: 45px 30px;
+}
+
+.social-image {
+  height: 23px;
+  width: 23px;
+  padding: 8px 8px;
+  padding-bottom: 2px;
+}
+
+.title {
+  color: #fff;
+  font-weight: 100;
+  font-size: 90px;
+  font-family: Yeseva One;
+  max-width: 970px;
+  margin: 0 auto;
+  text-align: center;
+  
+}
```

```suggestion
}

```
Add blank line in the end on the file